ccd-definition/CaseEventToFields/CareSupervision/changeCaseName.json

```diff
@@ -15,9 +15,9 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "changeCaseName",
     "CaseFieldID": "caseName",
-    "CaseEventFieldLabel": "*Case name",
+    "CaseEventFieldLabel": "Case name",
     "PageFieldDisplayOrder": 2,
-    "DisplayContext": "OPTIONAL",
+    "DisplayContext": "MANDATORY",
```

Comment:
Think changeCaseName event is deprecated, what is the point of updating this CaseEventToFields ?
Just asking in case anything is missing.

Reply:
have un-deprecated this one so keepin it around!